### PR TITLE
✨ Allow no query in creator and collector api

### PR DESCRIPTION
**Pull request by Tsai-Hsueh-Kuan**
✨ Allow no query in creator and collector api

### CODE REVIEW

**Review by nnkken — COMMENTED** [review]
No need to have new functions, simply modify the SQL in `GetCollector` and `GetCreators` by modifying the conditions using the `WHERE $1 = '' OR i.owner = $1` pattern

File: db/nft.go [hunk 1]
```diff
@@ -260,7 +260,8 @@ func GetCollector(conn *pgxpool.Conn, q QueryCollectorRequest) (res QueryCollect
 		FROM iscn as i
 		JOIN nft_class as c ON i.iscn_id_prefix = c.parent_iscn_id_prefix
 		JOIN nft AS n ON c.class_id = n.class_id
-		WHERE i.owner = $1
+			AND ($4::text[] IS NULL OR n.owner != ALL($4))
+		WHERE $1 = '' OR i.owner = $1
```
Comment (nnkken) [inline db/nft.go:264]: Are these 2 lines order reversed? Should it be something like below?
```
		WHERE ($1 = '' OR i.owner = $1)
			AND ($4::text[] IS NULL OR n.owner != ALL($4))
```

Comment (Tsai-Hsueh-Kuan) [reply db/nft.go:264]: I refer here: https://github.com/likecoin/likecoin-chain-tx-indexer/blob/22f9a79839dc4fda86e269418a259f28aa553925/db/nft.go#L81
Or should the order be reversed here?

Comment (nnkken) [reply db/nft.go:264]: Turns out the planner gives the same plan, so I guess it's OK

**Review by nnkken — APPROVED** [review]

**Pull request merged**